Extension/src/LanguageServer/settings.ts

```diff
@@ -414,337 +517,157 @@ export class CppSettings extends Settings {
+    public get vcFormatSpaceRemoveAroundUnaryOperator(): boolean { return this.getAsBoolean("vcFormat.space.removeAroundUnaryOperator"); }
     public get vcFormatSpaceAroundBinaryOperator(): string {
         // These strings have default values in package.json, so should never be undefined.
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        return super.Section.get<string>("vcFormat.space.aroundBinaryOperator")!;
+        return this.getAsString("vcFormat.space.aroundBinaryOperator")!;
     }
-
     public get vcFormatSpaceAroundAssignmentOperator(): string {
         // These strings have default values in package.json, so should never be undefined.
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        return super.Section.get<string>("vcFormat.space.aroundAssignmentOperator")!;
+        return this.getAsString("vcFormat.space.aroundAssignmentOperator")!;
     }
-
     public get vcFormatSpacePointerReferenceAlignment(): string {
         // These strings have default values in package.json, so should never be undefined.
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        return super.Section.get<string>("vcFormat.space.pointerReferenceAlignment")!;
+        return this.getAsString("vcFormat.space.pointerReferenceAlignment")!;
     }
-
     public get vcFormatSpaceAroundTernaryOperator(): string {
         // These strings have default values in package.json, so should never be undefined.
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        return super.Section.get<string>("vcFormat.space.aroundTernaryOperator")!;
+        return this.getAsString("vcFormat.space.aroundTernaryOperator")!;
     }
-
     public get vcFormatWrapPreserveBlocks(): string {
         // These strings have default values in package.json, so should never be undefined.
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        return super.Section.get<string>("vcFormat.wrap.preserveBlocks")!;
-    }
-
-    public get dimInactiveRegions(): boolean {
-        return super.Section.get<boolean>("dimInactiveRegions") === true
-            && this.intelliSenseEngine === "default"
-            && vscode.workspace.getConfiguration("workbench").get<string>("colorTheme") !== "Default High Contrast";
-    }
-
-    public get sshTargetsView(): string {
-        return super.Section.get<string>("sshTargetsView") ?? 'default';
+        return this.getAsString("vcFormat.wrap.preserveBlocks")!;
     }
+    public get dimInactiveRegions(): boolean { return this.getAsBoolean("dimInactiveRegions") && this.intelliSenseEngine === "default" && vscode.workspace.getConfiguration("workbench").get<string>("colorTheme") !== "Default High Contrast"; }
+    public get sshTargetsView(): string { return this.getAsString("sshTargetsView") ?? 'default'; }
```

Comment:
Is `default` already the default for this setting?  Can you remove the `?? 'default'`?